cluster/gce/util.sh

```diff
@@ -83,17 +83,20 @@ function set-linux-node-image() {
 # Requires:
 #   WINDOWS_NODE_OS_DISTRIBUTION
 # Sets:
-#   WINDOWS_NODE_IMAGE_PROJECT
```

Comment:
Why is the set-windows-node-image function being changed? I wouldn't expect that to be updated.